file-formats/rvbc/type/zif_aff_rvbc_v1.intf.abap

```diff
@@ -0,0 +1,122 @@
+"! AFF type for the object RVBC ( Review Booklets )
+INTERFACE zif_aff_rvbc_v1
+  PUBLIC.
+
+  TYPES ty_app_id             TYPE c LENGTH 70.
+  TYPES ty_ina1_service_name  TYPE c LENGTH 40.
```

Comment:
I am not sure. Is the service name really 40 characters long. Most objects only support 30 characters.